web/src/views/ProfileCreate.tsx

```diff
@@ -62,9 +64,22 @@ const ProfileCreate: React.FC = () => {
       // 4. Create Project Request
       await api.createProjectRequestByProfileId(profileId);
 
+      // 5. Invite technicalCOntact to bcgov repo, only invite in production environment
+      if (process.env.NODE_ENV === 'production') {
```

Comment:
i don't like this. Is this because the test and dev environments are hardcoded to the bcgov/bcdevops github orgs?

Reply:
could you not configure the app to invite users into 'test' organizations?